spec/factories/projects.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :project do
+    name 'MyString'
+    blurb 'MyText'
+    description 'MyText'
+  end
```

Comment:
We should add a `with_comments` trait and `with_collaborators` trait to optionally include comments and collaborators in the factory.